docker/Dockerfile

```diff
@@ -29,6 +29,9 @@ RUN git clone https://github.com/pytorch/pytorch
 ENV USE_CUDA "0"
 ENV USE_MPI "0"
 
+# Value for -D_GLIBCXX_USE_CXX11_ABI flag; if empty, it is infered from `torch._C`.
+ENV CXX_ABI "${cxx_abi}"
```

Comment:
lol, didn't saw it is being deleted in last pr

Reply:
I kicked off a test build with your trigger. Should we wait for it to finish before merging?